src/main/resources/logback.xml

```diff
@@ -15,23 +15,16 @@
         <target>System.out</target>
         <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
             <providers>
-                <timestamp>
-                    <fieldName>timestamp</fieldName>
-                </timestamp>
                 <loggerName/>
                 <logLevel>
                     <fieldName>severity</fieldName>
                 </logLevel>
-                <message/>
-                <stackHash/>
-                <stackTrace>
-                    <throwableConverter class="net.logstash.logback.stacktrace.ShortenedThrowableConverter">
-                        <maxDepthPerThrowable>30</maxDepthPerThrowable>
-                        <maxLength>2048</maxLength>
-                        <shortenedClassNameLength>20</shortenedClassNameLength>
-                        <rootCauseFirst>true</rootCauseFirst>
-                    </throwableConverter>
-                </stackTrace>
+                <message>
+                    <fieldName>shortMessage</fieldName>
+                </message>
+                <pattern>
+                    <pattern>{ "message": "%exception", "serviceContext": { "service": "fdk-public-service-harvester" } }</pattern>
```

Comment:
Er message ikke allerede i bruk. Bruke exception feltet?

Reply:
GCP krever at `message` er en stackTrace, for at den skal tolke noe fra loggen som en `ReportedErrorEvent`. Det faktiske `message`-feltet blir omdøpt til "shortMessage" vha `fieldName`.
se her: https://cloud.google.com/error-reporting/reference/rest/v1beta1/projects.events/report#ReportedErrorEvent

![Screenshot from 2021-07-02 07-47-12](https://user-images.githubusercontent.com/50194012/124226872-0e3d8200-db0a-11eb-82ae-0529d32eb260.png)